org-owners-guide.md

```diff
@@ -0,0 +1,66 @@
+# GitHub Organization Owners Guide
+
+The Kubernetes project leverages multiple GitHub organizations to store and
+organize code. This guide contains the details on how to run those organizations
+to for CNCF compliance and for the guidelines of the community.
+
+## Organization Naming
+
+Kubernetes managed organizations should be in the form of `kubernetes-[thing]`.
+For example, [kubernetes-client](https://github.com/kubernetes-client) where the
+API clients are housed.
+
+Prior to creating an organization please contact the steering committee for
+direction and approval.
+
+Note: The CNCF, as part of the Linux Foundation, holds the trademark on the
+Kubernetes name. All GitHub organizations with Kubernetes in the name should be
+managed by the Kubernetes project or use a different name.
+
+## Contributor License Agreement (CLA)
+
+All code contribute to the Kubernetes project needs to be done by those who are
+contributing under a signed CLA. This should be enforced with automation and bots.
+
+The automation is run by the Linux Foundation. When you have a new organization
+or repository setup that needs to have CLA checking added please contact
+[Ihor Dvoretskyi](https://github.com/idvoretskyi) and
+[Chris Aniszczyk](https://github.com/caniszczyk).
+
+## Code Of Conduct
+
+Every repository is required to list the code of conduct. The file name needs
+to be at the root of the repository and named `code-of-conduct.md`. That file
+will have a link to the Kubernetes community repository. This file in each
+repository should be [the file found in the Kubernetes repo](https://github.com/kubernetes/kubernetes/blob/master/code-of-conduct.md).
+
+## Software License
+
+All software that is part of the Kubernetes project needs to be licensed under
+the Apache 2 License.
+
+## Transferring Outside Code Into A Kubernetes Organization
+
+Due to licensing and CLA issues, prior to transferring software into a Kubernetes
+managed organization there is some due diligence that needs to occur. Please
+contact the steering committee and CNCF prior to moving any code in.
+
+It is easier to start new code in a Kubernetes organization than it is to
+transfer in existing code.
+
+## Current Organizations In Use
+
+The following organizations are currently known to be part of the Kubernetes
+project:
+
+* kubernetes
+* kubernetes-incubator
+* kubernetes-security
+* kubernetes-client
+* kubernetes-incubator-retired
+* kubernetes-helm
+* kubernetes-graveyard
+* kubernetes-csi
+
+This list is likely incomplete. If you are aware of additions please contribute
```

Comment:
I believe the list is still being hashed out elsewhere.

Reply:
We've since hashed this list out https://github.com/kubernetes/community/issues/1407#issuecomment-357114370